motr/pg.h

```diff
@@ -355,24 +355,30 @@ struct pargrp_iomap {
 	 * - number of columns = N.
 	 * Each element of matrix is worth PAGE_SIZE;
 	 * A unit size worth of data holds a contiguous chunk of file data.
-	 * The file offset grows vertically first and then to the next
-	 * data unit.
```

Comment:
Can we add a simple diagram here?
such as:
```

        If the PAGE_SIZE is 4k, and unit_size is 16k, for a file/object of 112k length,
        we have the following bufs, with corresponding offset.
            
        [0k       ][16k     ][32k     ][48k     ][64k     ][80k     ][96k     ]   
        [4k       ][20k     ][36k     ][52k     ][68k     ][84k     ][100k    ]   
        [8k       ][24k     ][40k     ][56k     ][72k     ][88k     ][104k    ]   
        [12k      ][28k     ][44k     ][60k     ][76k     ][92k     ][108k    ]   
```

```

Reply:
Yes, added.